easylist_adult/adult_thirdparty.txt

```diff
@@ -60,6 +62,7 @@
 ||creative.stripchat.com^
 ||creative.strpjmp.com^
 ||ddfcash.com^$third-party
+||dmzjmp.com/widgets/
```

Comment:
You can blacklist entirely, there are no god use of this domain at all
- https://mypdns.org/my-privacy-dns/matrix/-/issues/2092